cmd/knoxite/store.go

```diff
@@ -38,6 +38,7 @@ type StoreOptions struct {
 	Encryption       string
 	FailureTolerance uint
 	Excludes         []string
+	ExcludeExternalSymlinks bool
```

Comment:
Im not sure but shouldn't the types of the struct fields align after applying gofmt?

Reply:
Looks like my setup back than didn't include gofmt. Resolved.